bundles/http/Citrix/Microsoft-Outlook/script.js

```diff
@@ -1,230 +1,1198 @@
+        `v1.0/me/events`,
+        {
+            method: "POST",
+            body: JSON.stringify({
+
+                subject: actionParameters.subject,
+                body: {
+                    contentType: "HTML",
+                    content: actionParameters.content
+                },
+                start: {
+                    dateTime: actionParameters.startDateTime,
+                    timeZone: ""
+                },
+                end: {
+                    dateTime: actionParameters.endDateTime,
+                    timeZone: ""
+                },
+                location: {
+                    displayName: actionParameters.location
+                },
+                attendees: [{
+                    emailAddress: {
+                        address: actionParameters.email1
+                    },
+                    type: actionParameters.type1
+                },
+                {
+                    emailAddress: {
+                        address: actionParameters.email2
+                    },
+                    type: actionParameters.type2
+                }, {
+                    emailAddress: {
+                        address: actionParameters.email3
+                    },
+                    type: actionParameters.type3
+                },
+                {
+                    emailAddress: {
+                        address: actionParameters.email4
+                    },
+                    type: actionParameters.type4
+                },
+                {
+                    emailAddress: {
+                        address: actionParameters.email5
+                    },
+                    type: actionParameters.type5
+                },
+                {
+                    emailAddress: {
+                        address: actionParameters.email6
+                    },
+                    type: actionParameters.type6
+                }
+                ],
+                allowNewTimeProposals: true,
+                isOnlineMeeting: actionParameters.isOnlineMeeting,
+                onlineMeetingProvider: actionParameters.onlineMeetingProvider
+
+            }),
+        }
+    );
+    console.log(JSON.stringify(responseOfcreateOutlookOneTimeEventCurrentTimezone));
+    if (!responseOfcreateOutlookOneTimeEventCurrentTimezone.ok) {
+        throw new Error(`createOutlookOneTimeEventCurrentTimezone (${responseOfcreateOutlookOneTimeEventCurrentTimezone.status}: ${responseOfcreateOutlookOneTimeEventCurrentTimezone.statusText})`);
+    }
+    const user_id = [actionParameters.userId];
+    console.log(...user_id);
+    await calendarView(client, dataStore, user_id, startDate);
+    await myEvents(client, dataStore, user_id)
+}
+
+//service action for Create One Time Event with Custom Timezone
+async function createOutlookOneTimeEventCustomTimezone({
+    dataStore,
+    client,
+    actionParameters,
+}) {
+    const startDate = moment.utc().subtract(30, 'm').format();
```

Comment:
I believe the requirement is to subtract 1 min from the current time. Update the code accordingly.